lib/wallet/api/http/Cardano/Wallet/Api/Http/Server/Error.hs

```diff
@@ -553,6 +551,22 @@ instance IsServerError ErrBalanceTx where
                 , fmt $ blockListF' "-" conflictF conflicts
                 ]
         ErrBalanceTxOutputError err -> toServerError err
+        ErrBalanceTxUnableToCreateInput ->
```

Comment:
On second thought, I'm not sure about the notion of "creating inputs". We're not really "creating" inputs, we're just selecting or adding them.

Would `UnableToSelectInput` be better? Or perhaps `EmptyUTxO` wasn't that bad after all, even if there are cases where we _can_ balance with an empty utxo set? 🤔

Reply:
> On second thought, I'm not sure about the notion of "creating inputs". We're not really "creating" inputs, we're just selecting or adding them.

The way I rationalised it to myself was that:
- we _select_ existing UTxOs from the set of available UTxOs
- we _create_ inputs that reference those existing UTxOs 
 
> Would `UnableToSelectInput` be better? Or perhaps `EmptyUTxO` wasn't that bad after all, even if there are cases where we _can_ balance with an empty utxo set? 🤔

Out of these two, I'd be in favour of `UnableToSelectInput`.

But would be happy to discuss with you tomorrow! If we come to a new consensus, we can create a PR that updates the name.